script/release.py

```diff
@@ -65,7 +65,13 @@ def tag_and_generate_changelog(new_version_num):
 
 
 def upload_sdist(new_version_num):
-    call_bash_script('twine upload "dist/Appium-Python-Client-{}.tar.gz"'.format(new_version_num))
+    push_file = 'dist/Appium-Python-Client-{}.tar.gz'.format(new_version_num)
+    try:
+        call_bash_script('twine upload "{}"'.format(push_file))
+    except Exception as e:
+        print('Failed to upload {} to pypi. '
```

Comment:
should the error be printed to stderr?
should we rethrow the exception after print?

Reply:
I thought it was enough to print messages as stdout without stop/rethrow errors.
Current usage is showing messages on CLI and should not stop this by this error.